rllib/examples/documentation/replay_buffer_demo.py

```diff
@@ -0,0 +1,127 @@
+# Demonstration of RLlib's ReplayBuffer workflow
```

Comment:
Could we add this to BUILD as well? To run in the "RLlib documentation" job?
Look for the "documentation" tag and add it, e.g.:
```
py_test(
    name = "examples/documentation/custom_gym_env",
    main = "examples/documentation/custom_gym_env.py",
    tags = ["team:ml", "documentation"],
    size = "medium",
    srcs = ["examples/documentation/custom_gym_env.py"],
)

```